guillotina/contrib/workflows/utility.py

```diff
@@ -60,17 +60,18 @@ async def available_actions(self, request):
         def initial_state(self):
             return self._initial_state
 
-        async def do_action(self, action, comments):
+        async def do_action(self, action, comments, root=False):
```

Comment:
What do you think about renaming `root` to `bypass_permission_check` so it's more explicit?

Reply:
sure @vangheem much better